apps/camera_pipe/camera_pipe.cpp

```diff
@@ -280,7 +297,29 @@ Func process(Func raw, Type result_type,
     Expr out_height = processed.output_buffer().height();
 
     processed.bound(c, 0, 3); // bound color loop 0-3, properly
-    if (target.arch == Target::ARM) {
+    if (target.features_any_of({Target::HVX_64, Target::HVX_128})) {
+        const int tile_size = 128;
+        denoised.compute_at(processed, tx)
+            .vectorize(x, 64);
+        deinterleaved.compute_at(processed, tx)
+            .vectorize(x, 64)
+            .reorder(c, x, y)
+            .unroll(c);
+        corrected.compute_at(processed, tx)
+            .vectorize(x, 128)
+            .reorder(c, x, y)
+            .unroll(c);
+        processed.compute_root()
+            .hexagon()
+            .tile(x, y, tx, ty, xi, yi, tile_size, tile_size)
+            .reorder(xi, yi, c, tx, ty)
+            .parallel(ty);
+
+        // We can generate slightly better code if we know the output is a whole number of tiles.
+        processed
+            .bound(x, 0, (out_width/tile_size)*tile_size)
+            .bound(y, 0, (out_height/tile_size)*tile_size);
```

Comment:
Will you also be making a corresponding change to process.cpp to allocate the output to the same multiple of tile_size?  ...otherwise this may only compute a subset of the output (e.g. the remainder if it was not a multiple of tile_size)